pkgs/top-level/stage.nix

```diff
@@ -51,9 +79,23 @@ let
     };
 
   stdenvBootstappingAndPlatforms = self: super: {
-    stdenv = stdenv // { inherit platform; };
-    inherit buildPackages
-      system platform crossSystem;
+    inherit stdenv buildPackages
+      buildPlatform hostPlatform targetPlatform;
+  };
+
+  # The old identifiers for cross-compiling. These should eventually be removed,
```

Comment:
Can you open an issue to track the eventual removal? Perhaps after the next release we can add a deprecation message with `builtins.trace`